src/shared/meson.build

```diff
@@ -173,6 +173,11 @@ shared_sources = files('''
         net-condition.h
         netif-naming-scheme.c
         netif-naming-scheme.h
+        network-cloud-azure.c
```

Comment:
Wouldn't ```src/libsystemd-network``` be a better location for the new files? After all they are specific to networkd. That way we avoid a dependency on libcurl for the whole of systemd.

Reply:
Actually libcurl dependency is removed from here now. Only linked with networkctl and later will be linked with networkd